src/app/components/MixerCard/MixerCard.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Typography from '../Typography/Typography';
+import style from './MixerCard.module.css';
+
+export default function MixerCard(): JSX.Element {
```

Comment:
This still needs props to pass information to `Typography` instead of having it fixed.

Reply:
Resolved in [00195ad](https://github.com/Seborider/plugs./pull/31/commits/00195ad221925d0cfcd5f0597917ef5ff3c36e94)